minishift-ci-index.yaml

```diff
@@ -363,7 +363,7 @@
           git_repo: minishift
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
-          timeout: '300m'
+          timeout: '390m'
```

Comment:
@amitkrout, why you changed this, since now release job, is actually same as PR job and that doesn't timeout.

Reply:
Ahh..i misunderstood with nightly job. Thanks for pointing it out. Will revert it back

Reply:
Done